app/services/locations_csv.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class LocationsCSV < CsvGenerator
+  def initialize(record_or_records)
+    @records = Array(record_or_records).map { |record| LocationWithAddress.new(record) }
+  end
+
+  def attributes
+    %w(uid title address_line_1 address_line_2 address_line_3 town
```

Comment:
Can we format these with each successive attribute on a newline? As per the styling here https://github.com/guidance-guarantee-programme/location-register/pull/31/files#diff-6269b912e824598cdd2e7528fdd8e17aR12